packages/ui/src/types/authenticator/form.ts

```diff
@@ -20,21 +22,21 @@ export type FormFieldComponents =
 /**
  * Used to customize form field attributes for each authenticator screen.
  */
-export type FormFields = {
-  [key in FormFieldComponents]?: FormField;
+export type AuthFormFields = {
+  [key in FormFieldComponents]?: FormFields;
 };
 
 /**
  * Override option for each screen. Maps each input to override options.
  */
-export interface FormField {
-  [key: string]: FormFieldOptions;
+export interface FormFields {
+  [key: string]: FormField;
 }
```

Comment:
Made some naming changes -- this is really a collection of `FormField`s, so made it plural.

Reply:
👍